frontend/pages/hosts/details/cards/User/User.tsx

```diff
@@ -55,6 +55,10 @@ const User = ({
   const showGroups = showUsername && userNameDisplayValues.length > 0;
   const showChromeProfiles = chromeProfilesDisplayValues.length > 0;
   const showOtherEmails = otherEmailsDisplayValues.length > 0;
+  const userDepartment = [];
+  if (endUser && endUser.idp_department && endUser.idp_department !== "") {
```

Comment:
Just a nit - you can collapse this whole condition into `if (endUser?.idp_department)`

Reply:
Or even have the whole code block be `const userDepartments = endUser?.idp_department ?
[endUser.idp_department] : [];`

Reply:
Thanks! Fixed.